order-service/order-service.ts

```diff
@@ -0,0 +1,201 @@
+import { Tx, Pair, Position, OrderConfig } from '../types/common.js';
+import {
+  OrderSide,
+  OrderType,
+  BECH32_PREFIX,
+  LocalWallet,
+  SubaccountClient,
+  Network,
+  CompositeClient,
+  IndexerClient,
+} from '@dydxprotocol/v4-client-js';
+
+// order factory function
+const createOrderService = async (mnemonic: string, network: Network) => {
+  const wallet = await LocalWallet.fromMnemonic(mnemonic, BECH32_PREFIX);
+  const subaccount = new SubaccountClient(wallet, 0);
```

Comment:
Can't we work with the wallet itself instead of say, passing one of its subaccounts to order functions? I'm just curious and not stating that's the case.

Reply:
I didn't mean this change. The wallet should be created inside of the factory, not by the user.